openg2p_registration/models/openg2p_registration.py

```diff
@@ -346,17 +370,93 @@ def create_beneficiary_from_registration(self):
             'context': context
         }
 
+    @api.onchange('stage_id')
+    def checking_duplicates(self):
+        if int(self.stage_id.id) == 2:
+            my_list = self.search_beneficiary()
+            my_list = json.loads(my_list)
+            # print(my_list)
+            if my_list:
+                benf_ids = [li['beneficiary'] for li in my_list]
+                # print(benf_ids)
+                for i in benf_ids:
+                    self.update({'duplicate_beneficiaries_ids': (i)
```

Comment:
The error sent over Whatsapp is coming from here. duplicate_beneficiaries_ids is raising the exception while saving the data. Since it's a [Many2Many](https://github.com/OpenG2P/openg2p-erp/pull/70/files#diff-bb7cdb4faf14364acea433a98ef5a0e3893592e0bd3031382438f5aa7e5671deR171) field you would need special operators as described in http://www.odooninja.com/update-many2many-field/

In your case, it's going to be (6, 0, {duplicate ids}), such as self.update({'duplicate_beneficiaries_ids': [( 6, 0, {duplicate ids})]})